selfdrive/car/hyundai/carstate.py

```diff
@@ -145,6 +163,7 @@ def __init__(self, CP):
     self.left_blinker_flash = 0
     self.right_blinker_on = 0
     self.right_blinker_flash = 0
+    self.no_radar = self.CP.carFingerprint in FEATURES["non_scc"]
```

Comment:
We won't upstream this, either a car uses stock adaptive cruise control or openpilot's, but no ACC isn't an option. It violates the fully on fully off nature of openpilot.

Reply:
Ok I will remove all non_scc cars then

Reply:
removed